github-dark.css

```diff
@@ -2782,7 +2782,10 @@
   .jump-to-suggestions-results-container mark, .dropdown-menu a:hover,
   .jump-to-suggestions-results-container [aria-selected="true"] .jump-to-suggestions-path,
   .jump-to-suggestions-results-container [aria-selected="true"] .jump-to-octicon,
-  .markdown-body strong, .markdown-format strong, .text-gamma {
+  .markdown-body strong, .markdown-format strong, .text-gamma,
+  .select-menu-item[aria-selected="true"],
+  .select-menu-item[aria-selected="true"].selected {
+   {
```

Comment:
Please remove extra `{`.

Reply:
how the heck did that get in there... Im going blinder :)

Wow new design extra weird ;) now on this portion... ugh.

![capture](https://user-images.githubusercontent.com/31389848/41692326-966fb44e-74f7-11e8-8c37-ed67052ecfff.PNG)

Reply:
Done ;)

Reply:
Yeah, the brown background is supposed to highlight a review comment... I think we added it because Refined GitHub was changing the color, so we had to override it.